src/parser/monk/mistweaver/modules/features/MasteryStats.tsx

```diff
@@ -29,6 +30,18 @@ class MasteryStats extends Analyzer {
     expelHarm: ExpelHarm,
   };
 
+  protected essenceFontMastery!: EssenceFontMastery;
+  protected envelopingMists!: EnvelopingMists;
+  protected soothingMist!: SoothingMist; 
+  protected renewingMist!: RenewingMist;
+  protected vivify!: Vivify;
+  protected expelHarm!: ExpelHarm;
+
+  constructor(args: Options) {
+    super(args);
+    this.addEventListener(Events.fightend, this.endFight);
```

Comment:
`debug && this.addEventListener(...)`

Then rename the function to `fightEndDebug`, and you can remove the `if(debug)` in there 😎